cartographer_ros/cartographer_ros/pbstream_map_publisher_main.cc

```diff
@@ -82,8 +85,9 @@ std::unique_ptr<nav_msgs::OccupancyGrid> LoadOccupancyGridMsg(
 
 void Run(const std::string& pbstream_filename, const std::string& map_topic,
          const std::string& map_frame_id, const double resolution) {
-  std::unique_ptr<nav_msgs::OccupancyGrid> msg_ptr =
-      LoadOccupancyGridMsg(pbstream_filename, resolution);
+  ::cartographer::mapping::ValueConversionTables conversion_lookup_tables;
```

Comment:
This could've been moved into the function LoadOccupancyGridMsg, no extra function parameter needed.